### PR TITLE
fix: Fixed wrong event data on page view event

### PR DESCRIPTION
**Pull request by vishnu-gp**
## Description

Fixed the wrong event data being sent for page view events. 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Manually

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by AnaghHegde — APPROVED** [review]

File: app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/NewActionServiceCEImpl.java [hunk 1]
```diff
@@ -1117,7 +1118,7 @@ private Mono<ActionExecutionRequest> sendExecuteAnalyticsEvent(
                             ),
                             "orgId", application.getWorkspaceId(),
                             "appId", action.getApplicationId(),
-                            "appMode", TRUE.equals(viewMode) ? ApplicationMode.PUBLISHED.toString() : ApplicationMode.EDIT.toString(),
+                            "appMode", appMode,
```
Comment (AnaghHegde) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/NewActionServiceCEImpl.java:1121]: Minor: Can we please remove the hardcoded string?

**Review by AnaghHegde — CHANGES_REQUESTED** [review]

File: app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/NewActionServiceCEImpl.java (hunk 1 above)
Comment (vishnu-gp) [reply app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/NewActionServiceCEImpl.java]: Done @AnaghHegde

**Pull request merged**